multicluster/cmd/multicluster-controller/options.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/spf13/pflag"
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/klog/v2"
+	ctrl "sigs.k8s.io/controller-runtime"
+
+	mcsv1alpha1 "antrea.io/antrea/multicluster/apis/multicluster/v1alpha1"
+)
+
+type Options struct {
+	// The path of configuration file.
+	configFile     string
+	SelfSignedCert bool
+	options        ctrl.Options
+	leader         bool
+	member         bool
+}
+
+func newOptions() *Options {
+	return &Options{
+		// TODO: remove cert-manager dependency and
+		// use self signed cert in the future
+		SelfSignedCert: false,
```

Comment:
this need to be `true` after cert-manager dependency is removed, since the changes is already in feature branch, Could you help to remove the comment and set it as `true`, thanks,

Reply:
Ok sure